aws_topology/stackstate_checks/aws_topology/flowlogs.py

```diff
@@ -80,27 +77,35 @@ def add_traffic(self, start, end, traffic_type, incoming, byte_count, nwitf, rev
         if self.end_time == 0 or end > self.end_time:
             self.end_time = end
         if incoming ^ reverse:
-            self.bytes_received += byte_count
+            self.half_bytes_received += byte_count
         else:
-            self.bytes_sent += byte_count
+            self.half_bytes_sent += byte_count
         if str(traffic_type) != self.traffic_type:
             self.traffic_type = "unknown"
 
     @property
     def interval_seconds(self):
         return self.end_time - self.start_time
 
+    @property
+    def total_bytes_sent(self):
+        return self.half_bytes_sent / 2
```

Comment:
I'm confused here. So `total_bytes` are half of `half_bytes`? So one quater of bytes?
We really need a comment with explanation what is happening here...